setup.py

```diff
@@ -828,6 +828,7 @@ def print_box(msg):
                 'include/torch/csrc/api/include/torch/nn/modules/*.h',
                 'include/torch/csrc/api/include/torch/nn/modules/container/*.h',
                 'include/torch/csrc/api/include/torch/nn/parallel/*.h',
+                'include/torch/csrc/api/include/torch/nn/utils/*.h',
```

Comment:
I'm not too familiar with the process for declaring dependencies in `setup.py`, but is there an easier way to do this than have to manually include new header files when adding new directories in the C++ API? Could we possibly find these header files using `glob` and have them added here automatically?